luigi/interface.py

```diff
@@ -79,7 +80,7 @@ class EnvironmentParamsContainer(task.Task):
         is_global=True, default=False,
         description='Ignore if similar process is already running')
     lock_pid_dir = parameter.Parameter(
-        is_global=True, default='/var/tmp/luigi',
+        is_global=True, default=os.path.join(tempfile.gettempdir(), 'luigi'),
```

Comment:
What? Where's the code logic that changes it to `/var/tmp` if `/tmp` is not available.

> Personally I would expect people to explicitly specify lock_pid_dir the configuration if they rely on its location. 

Yes.

Also, I read up a little bit on /tmp vs /var/tmp [1]. And it seems like `/tmp` is actually what we want here.

http://www.google.se/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&ved=0CCIQFjAA&url=http%3A%2F%2Flinuxers.org%2Farticle%2Fdifferences-between-tmp-and-vartmp&ei=HvPAVInWNMviywOUgIH4CQ&usg=AFQjCNE7JtC--WEDeW9R5D6CaJdcmDhdFg&bvm=bv.83829542,d.bGQ